gems/quilt_rails/app/controllers/quilt/react_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Quilt
+  class ReactController < ApplicationController
```

Comment:
Do we need to inherit from the `ApplicationController`? Are we expecting people to add behavior to this controller by changing the `ApplicationController`, or we want it to be totally in independent? One way to think about it is: if people add an authentication in front of all controllers, should this one also be protected?

Reply:
I'd think it should be

Reply:
👍. Just confirming